packages/issues-fetcher/src/index.ts

```diff
@@ -0,0 +1,142 @@
+import type { Model, ModelIdsByProvider, ProviderNames } from '@proj-airi/jem'
+import * as fs from 'node:fs'
+import path from 'node:path'
+import * as process from 'node:process'
+import { models } from '@proj-airi/jem'
+
+import git from 'isomorphic-git'
+import { Octokit } from 'octokit'
+import { parseModelIssue } from './issue-parser.ts'
+import { createRequire } from 'node:module'
+
+const http = createRequire(import.meta.url)('isomorphic-git/http/node')
+const gitUrl = new URL('https://github.com/moeru-ai/inventory.git')
+gitUrl.password = process.env.GITHUB_TOKEN!
+gitUrl.username = process.env.GITHUB_USERNAME!
+
+const rootDir = path.join(process.cwd(), '..', '..')
+const modelsFilePath = path.join(rootDir, 'packages', 'jem', 'src', 'models.ts')
+
+function generateModelsFileContent(models: Model<ProviderNames, ModelIdsByProvider<ProviderNames>>[]) {
+  return `// Auto-generated file. Do not edit.
+
+  import type { Model } from './types'
+
+  export const models = ${JSON.stringify(models, null, 2)} as const satisfies Model[]
+  `
```

Comment:
Generated file doesn't have a final new line...Let me fix it later.

Reply:
```suggestion
`
```

Maybe we just need to clear the ` ` spaces?

Reply:
Feel free to try to use eslint for formatting helpers.